newIDE/app/src/InstancesEditor/index.js

```diff
@@ -804,26 +833,38 @@ export default class InstancesEditor extends Component<Props> {
       this.highlightedInstance.setInstance(null);
   };
 
+  _onUpInstance = (
+    instance: gdInitialInstance,
+    sceneX: number,
+    sceneY: number
+  ) => {
+    if (!this.hasCursorMovedSinceItIsDown) {
+      this.props.instancesSelection.selectInstance({
+        instance,
+        multiSelect: this.keyboardShortcuts.shouldMultiSelect(),
+        layersLocks: this._getLayersLocks(),
+      });
+      if (this.props.onInstancesSelected) {
+        this.props.onInstancesSelected(
+          this.props.instancesSelection.getSelectedInstances()
+        );
+      }
+
+      if (this.selectionRectangle.hasStartedSelectionRectangle()) {
```

Comment:
How can a selection rectangle could have started if the cursor has not moved since it's down?

Reply:
It start when the mouse is pressed, so a rectangle of 1x1 pixel is done when the mouse does just a click but it doesn't have any side effect as it can't select anything.